phalcon/db/column.zep

```diff
@@ -142,6 +142,11 @@ class Column implements ColumnInterface
 	const TYPE_TIMESTAMP = 17;
 
 	/**
+	 *  Small integer abstract type
+	 */
+	const TYPE_SMALLINTEGER = 18;
```

Comment:
We already have a type TYPE_BIGINTEGER. It would be very strange (though quite in the spirit of PHP 😆) to use the name TYPE_SMALLINT.